app/git/git.go

```diff
@@ -298,3 +299,20 @@ func NewRelease(repoName string, tagName string, releaseName string, body string
 	logger.Info.Printf("added release '%s', tagged '%s' to repo '%s'", releaseName, tagName, repoName)
 	return nil
 }
+
+// ValidateUser checks if the provided GitHub username exists.
+// Returns nil if valid, or an error if invalid.
+func ValidateUser(username string) error {
```

Comment:
GitHub might return an error for various kind of reasons. Instead I would suggest to change the name to sth. like: `DoesAccountExist` and the return type to `(bool, error)`